packages/translations/src/locales/ca-ES/calendar.ts

```diff
@@ -0,0 +1,18 @@
+import { CalendarTranslations } from '../../types/calendar.translations'
+
+export const calendarCaES: CalendarTranslations = {
+  Today: 'Avui',
+  Month: 'Mes',
+  Week: 'Setmana',
+  Day: 'Dia',
+  events: 'Events',
```

Comment:
I don't know Catalan. But is this one right?

"Esdeveniment" = Event
"Events" = Events
?